app/controllers/v3/service_credential_bindings_controller.rb

```diff
@@ -205,6 +206,16 @@ def enqueue_unbind_job(type, binding_guid)
     pollable_job.guid
   end
 
+  def check_parameters_support(service_instance, message)
+    unless service_instance.is_a?(VCAP::CloudController::ManagedServiceInstance)
```

Comment:
Being that the error message in the exception thrown is specific for user-provided service instances, and that's the only other ServiceInstance type we have, I think this would be more readable if it was:
```
if service_instance.is_a?(VCAP::CloudController::UserProvidedServiceInstance) && message.requested?(:parameters)
       raise exception here
end
```